src/main/kotlin/mate/academy/Lottery.kt

```diff
@@ -0,0 +1,15 @@
+package mate.academy
+
+import kotlin.random.Random
+
+class Lottery : ColorSupplier() {
```

Comment:
This class can be signleton, so we won't need to create new instance each time to get random ball:
https://www.baeldung.com/kotlin/singleton-classes